superwall/src/main/java/com/superwall/sdk/config/ConfigManager.kt

```diff
@@ -393,4 +400,34 @@ open class ConfigManager(
                 fetchDuration = System.currentTimeMillis() - startTime,
             )
     }
+
+    suspend fun checkForWebEntitlements() {
+        ioScope.launch {
+            webPaywallRedeemer().redeem(WebPaywallRedeemer.RedeemType.Existing)
+            if (entitlements.all.size != entitlements.active.size) {
+                // This runs only if user does not have all of the entitlements
+                webPaywallRedeemer()
+                    .checkForWebEntitlements(
+                        UserId(Superwall.instance.userId),
+                        DeviceVendorId(Superwall.instance.vendorId),
+                    ).fold(onSuccess = { webEntitlements ->
+
+                        if (webEntitlements.isNotEmpty()) {
```

Comment:
Again if the web entitlements have gone this won't update the subs status to reflect that. Why not call pollEntitlements instead of having this separate func?